### ISSUE TITLE
reqwest-0.10.0 native_tls_crate not found

### ISSUE THREAD

**Issue by andir** [kolloch/crate2nix#62]
For some reason we fail to build `reqwest-0.10.0` with the following error.

```
$ nix-build -A reqwest
these derivations will be built:
  /nix/store/chndfc9rs3zbxcka8cc557l2czgk0pnd-rust_reqwest-0.10.0.drv
  /nix/store/mr4b5yg3y1zb8hn249izjwz7axy6drfg-rust_c2nix-reqwest-0.1.0.drv
building '/nix/store/chndfc9rs3zbxcka8cc557l2czgk0pnd-rust_reqwest-0.10.0.drv'...
unpacking sources
unpacking source archive /nix/store/qkr2c2yvlid23a83yk45kmmjmrgxvfv2-reqwest-0.10.0.tar.gz
source root is reqwest-0.10.0
setting SOURCE_DATE_EPOCH to timestamp 1577730381 of file reqwest-0.10.0/Cargo.toml
patching sources
configuring
building
Building src/lib.rs (reqwest)
Running rustc --crate-name reqwest src/lib.rs --out-dir target/lib --emit=dep-info,link -L dependency=target/deps --cap-lints allow -C opt-level=3 -C codegen-units=8 --extern base64=/nix/store/x7jafx876ai6094fm0dzpihanwrna5id-rust_base64-0.11.0-lib/lib/libbase64-072630678c.rlib --extern bytes=/nix/store/39hc1mb8fmlgf4
kxj63cni41lylqnb38-rust_bytes-0.5.3-lib/lib/libbytes-f73f885952.rlib --extern encoding_rs=/nix/store/7aknr8l90a2q6bz8wzxfknl30jdqkc1j-rust_encoding_rs-0.8.22-lib/lib/libencoding_rs-b17d0b6c7b.rlib --extern futures_core=/nix/store/vl08y9432b4cp1j07f79xpbqx61ww8dn-rust_futures-core-0.3.1-lib/lib/libfutures_core-857dc662
e2.rlib --extern futures_util=/nix/store/gli5vv0zjgkzwjyblv40aw8i7v66765r-rust_futures-util-0.3.1-lib/lib/libfutures_util-cb7c926876.rlib --extern http=/nix/store/h9cpmiyy4x4ksji0g9rbgrjffr9ihmnn-rust_http-0.2.0-lib/lib/libhttp-b50d8ba548.rlib --extern http_body=/nix/store/npcmn2fa1y6p2yly12h3jhlwmp61022z-rust_http-bo
dy-0.3.1-lib/lib/libhttp_body-284aab6c33.rlib --extern hyper=/nix/store/7k3nkwszf50a13qrxz7m2s7kr20qwr5k-rust_hyper-0.13.1-lib/lib/libhyper-1b7d7812b6.rlib --extern hyper_tls=/nix/store/6vhrih9z076zz6y0sdvgadxys1c0cgkk-rust_hyper-tls-0.4.0-lib/lib/libhyper_tls-b5fe24eba3.rlib --extern lazy_static=/nix/store/vixvw26wxb
mv92yhi9ksv4sxp9fkciqc-rust_lazy_static-1.4.0-lib/lib/liblazy_static-28e1bf1896.rlib --extern log=/nix/store/hvv6bj1aabyh2gyx7pn1z9pyvz0vklvk-rust_log-0.4.8-lib/lib/liblog-b1c5c93ca2.rlib --extern mime=/nix/store/0mml7gm35pgq321k2cigwky2wgfnpw4w-rust_mime-0.3.15-lib/lib/libmime-646e49dbcc.rlib --extern mime_guess=/nix
/store/vi5cj30kzyj9l9r8mwqwp91fmnyqy11i-rust_mime_guess-2.0.1-lib/lib/libmime_guess-6aa1218eeb.rlib --extern percent_encoding=/nix/store/jpdq03lsg0lfl8scbzhilff4r9a1xnkp-rust_percent-encoding-2.1.0-lib/lib/libpercent_encoding-162836dfd5.rlib --extern pin_project_lite=/nix/store/04n3yk0lawm9alfpn3my37s9slya90dv-rust_pi
n-project-lite-0.1.2-lib/lib/libpin_project_lite-3e6d52a4e8.rlib --extern serde=/nix/store/ikah92sbxzijlg3l8lazz34s395nlbj6-rust_serde-1.0.104-lib/lib/libserde-f8f832bfc5.rlib --extern serde_json=/nix/store/8f3fk3k6p6acylczzdk3vc4zwplrygzd-rust_serde_json-1.0.44-lib/lib/libserde_json-aa79c7c1bf.rlib --extern serde_url
encoded=/nix/store/lchfbic3lh38gkdjs2q8rcxnivbmi8rg-rust_serde_urlencoded-0.6.1-lib/lib/libserde_urlencoded-244bbb741c.rlib --extern time=/nix/store/kkp22brma6ab1v8wl8zchqlhxlzlhs5a-rust_time-0.1.42-lib/lib/libtime-d5aad28549.rlib --extern tokio=/nix/store/9wqjx42f9aqxw9i3prczfninigflgzgq-rust_tokio-0.2.6-lib/lib/libt
okio-cac0becf00.rlib --extern tokio_tls=/nix/store/5paxl4lwxhv2a47pbaw6crnkg96j3kwl-rust_tokio-tls-0.3.0-lib/lib/libtokio_tls-7fe734d4f7.rlib --extern url=/nix/store/nnaddhg9g57npg44kd3hjw6ssh3ia8kp-rust_url-2.1.0-lib/lib/liburl-c895737533.rlib --cfg feature="__tls" --cfg feature="blocking" --cfg feature="default" --c
fg feature="default-tls" --cfg feature="futures-util/io" --cfg feature="hyper-tls" --cfg feature="json" --cfg feature="native-tls-crate" --cfg feature="serde_json" --cfg feature="tokio-tls" --cfg feature="tokio/rt-core" --cfg feature="tokio/rt-threaded" --cfg feature="tokio/sync" --edition 2018 -C metadata=c817bbaedd 
-C extra-filename=-c817bbaedd --crate-type lib -L native=/nix/store/kra6g7g1wz6dvnj1b5j3dhsvfgxz32zp-openssl-1.1.1d/lib -l crypto -l ssl --color always
error[E0432]: unresolved import `native_tls_crate`
  --> src/async_impl/client.rs:18:5
   |
18 | use native_tls_crate::TlsConnector;
   |     ^^^^^^^^^^^^^^^^ use of undeclared type or module `native_tls_crate`

error[E0432]: unresolved import `native_tls_crate`
 --> src/connect.rs:7:5
  |
7 | use native_tls_crate::{TlsConnector, TlsConnectorBuilder};
  |     ^^^^^^^^^^^^^^^^ use of undeclared type or module `native_tls_crate`

error[E0433]: failed to resolve: use of undeclared type or module `native_tls_crate`
  --> src/tls.rs:11:13
   |
11 |     native: native_tls_crate::Certificate,
   |             ^^^^^^^^^^^^^^^^ use of undeclared type or module `native_tls_crate`

error[E0433]: failed to resolve: use of undeclared type or module `native_tls_crate`
  --> src/tls.rs:62:21
   |
62 |             native: native_tls_crate::Certificate::from_der(der).map_err(crate::error::builder)?,
   |                     ^^^^^^^^^^^^^^^^ use of undeclared type or module `native_tls_crate`

error[E0433]: failed to resolve: use of undeclared type or module `native_tls_crate`
  --> src/tls.rs:87:21
   |
87 |             native: native_tls_crate::Certificate::from_pem(pem).map_err(crate::error::builder)?,
   |                     ^^^^^^^^^^^^^^^^ use of undeclared type or module `native_tls_crate`

error[E0433]: failed to resolve: use of undeclared type or module `native_tls_crate`
  --> src/tls.rs:94:53
   |
94 |     pub(crate) fn add_to_native_tls(self, tls: &mut native_tls_crate::TlsConnectorBuilder) {
   |                                                     ^^^^^^^^^^^^^^^^ use of undeclared type or module `native_tls_crate`

error: aborting due to 6 previous errors

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
builder for '/nix/store/chndfc9rs3zbxcka8cc557l2czgk0pnd-rust_reqwest-0.10.0.drv' failed with exit code 1
cannot build derivation '/nix/store/mr4b5yg3y1zb8hn249izjwz7axy6drfg-rust_c2nix-reqwest-0.1.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/mr4b5yg3y1zb8hn249izjwz7axy6drfg-rust_c2nix-reqwest-0.1.0.drv' failed
```

It smells like we are not properly renaming a crate to `native_tls_crate`.

I crated a minimal test example here: https://github.com/andir/crate2nix-issues/tree/bcdc48383c091692865eff693166feb6749bdd96

**Comment by kolloch** [comment]
This is fixed by #64, correct?

**Comment by andir** [comment]
Yes
